### ISSUE TITLE
a feature request for once: can we have Bee Day

### ISSUE THREAD

**Issue by dinnerwoerror** [WaspStation/WaspStation-1.0#207]
## Name of feature:
every Wednesday will be Bee Day

### Related Pull Request on other codebase:
https://wiki.ss13.co/Ass_Day

![image](https://user-images.githubusercontent.com/58999300/81902260-ad08c700-9574-11ea-9fe8-d600c755ea5b.png)

### Additional Info:
the day admins are required to not talk to each other, half of them ban for building a mech as a roboticist, and the other half doesn't even ban plasma flooders

there will be some guidelines though

1. no griefing the **server**
2. no erp
3. no blowing up arrivals
4. you still have to respect the admins so if they tell you to stop you have to
5. no you arent getting an antag token for today

there will be many fun caveats about bee day

you will be allowed to grief and do stuff you'd normally do as an antag, so yes you are allowed to 'test' your toxins bomb in the security hallway and will therefore be allowed to ignore your laws as a silly con

security is allowed to be however brutal or lenient as they want but dont be surprised if le admin steps in during wardens murder obning

metagaming is encouraged so join a discord call

metagrudging is encouraged so behead nyaski reen for something they did in a previous round

IC in OOC is encouraged so screech about the traitor that capped you in robotics maint in the OOC chat

OOC will always be on unless the admins say it isn't, because griefing the server itself still is against the rules

**Comment by TheObserver-sys** [comment]
> metagrudging is encouraged so behead nyaski reen for something they did in a previous round
Now this is pod racing.

**Comment by miniusAreas** [comment]
Why the hell would we *promote* shittery on our server. If you miss this kind of stuff go back to Bee. This isnt where you put feature requests anyways.

**Comment by dinnerwoerror** [comment]
we're promoting shittery so that we may maintain a balanced audience of roleplayers and tryhards, so that no one side gains too much influence and we befall the role of what happened to Citadel station

and no, i'm not going back to bee station, because bee station is a ghetto for griefers and noobs, and the administration there is a fucking joke, like, seriously, people cannot truly explain in words how bad the admins there are, they're about the same tier as the moderation at TwitchTV, they have the MRP ruleset on the LRP server because someone thought it would look nice, thanks to how oblique and backhanded their rules regarding powergaming are, you gotta fucking walk on eggshells the entire time

and if this isn't where i'm supposed to put feature requests anyways then what in the seven hells is this for 

![Screenshot (582)](https://user-images.githubusercontent.com/58999300/82350596-68d05900-99b0-11ea-9f5f-26f064deb59b.png)

in short, an honorable attempt, now you may peacefully return to your profession as the chief bearer of cackle fruit

**Comment by Maldaris** [comment]
So couple things:

1) Straight up mugging Bee for their meme event isn't gonna happen. Either we come up with our own variant that fits our community and its playerbase, or we deal. (and dealing means @miniusAreas pretty spectacular events, so what's the problem?)
2) Yes, this is where feature requests go, but this isn't a feature request. This would not touch code, at all, and just generates noise for our developers. This is at best something run by the admins, and at worst, something organized by the community at large. 
3) Excuse you sir, but _I'm_ the chief bearer of the cackle fruit.

Closing this issue. Bring it up with me in discord.

**Comment by SomeAngryMiner** [comment]
even though i know this is for a meme day, i found that comment slightly insulting, owch.

**Comment by SynnGraffkin** [comment]
@Maldaris actually close the issue then, nerd.

**Comment by Maldaris** [comment]
@SynnGraffkin I would if I could, nerd.

**Comment by dinnerwoerror** [comment]
damn this got heated out of nowhere

**Comment by SynnGraffkin** [comment]
Maldaris with the capital M for Moron. 😏 

I like to poke fun at him, but yeah, this is more of something to run by admins / headmins in terms of policy, not really a code feature.

**Comment by dinnerwoerror** [comment]
gotta start somewhere baby
